### PR TITLE
pom.xml updates to support issue #27

### PR DESCRIPTION
**Pull request by phillipross**
pom.xml updates to support issue #27

### CODE REVIEW

**Comment by phillipross** [comment]
@dekobon This PR contains commits that reorganize the elements in the pom.xml and change line indentation from 2 to 4 spaces.  I dislike making large formatting changes like this since the diff is almost unreadable, so I definitely make it a habit to avoid doing this, but this is a one of those situation where it's necessary to get things maintainable :innocent:

File: pom.xml [hunk 1]
```diff
@@ -1,339 +1,386 @@
...
+            <url>yunong.ca</url>
+            <id>yunong</id>
+        </developer>
+        <developer>
+            <email>elijah@zupancic.name</email>
+            <name>Elijah Zuoancic</name>
+            <organization>Joyent</organization>
+            <organizationUrl>joyent.com</organizationUrl>
+        </developer>
+    </developers>
```
Comment (dekobon) [inline pom.xml]: You should add yourself ( @phillipross ) to the list of developers.

**Comment by dekobon** [comment]
Thanks for the PR. Will the steps for releasing change? If so, we may want to update the release section of the README.md.

Also, I don't mind the formatting changes - especially when we start to establish ownership of the project. In fact, I wouldn't mind reformatting everything to bring it closer to the Java standard conventions, but that isn't a big priority to me.

**Comment by dekobon** [comment]
Merge when ready!

**Comment by phillipross** [comment]
**Re: the release process**
I'm not sure about changes to the existing release process.  When I push to sonatype for other projects, I don't use sonatype's plugin, but rather I use the maven release and deployment plugins directly, so I don't know what might be different when using sonatype's plugin.  I think the instructions are exactly the same.  I didn't change any of the plugins configuration or make any changes to the profiles that are setup, so it should still work.  If you can add my sonatype id (philross) to the repo access list I'll be able to test things like this as well as push snapshots and releases.

**Re: conventions**
I also left the checkstyle plugin configuration in there, but set the configuration to skip the checkstyle process when I saw that the existing codebase didn't follow whatever rules have been defined.  I'd also like the codebase to be closer to standard conventions, and I think doing it sooner than later would be best.  I'll look into this next.

**Re: the merge**
I don't have write access to this repo to do the merge.  If you can followup with the necessary people to get me access I'd be grateful.

File: pom.xml (hunk 1 above)
Comment (phillipross) [inline pom.xml]: Oh, yes.  We should also probably move Yunong from developers section to a contributors section.  I believe the convention is that developers actively communicate and field questions to the community about the project, whereas contributors are people who should not be contracts directly.  If he's no longer actively involved he should be moved to a contributors section.

Comment (dekobon) [inline pom.xml]: +1

On Mon, Nov 2, 2015 at 3:29 PM, Phillip Ross <notifications@github.com>
wrote:

> In pom.xml
> <https://github.com/joyent/java-manta/pull/28#discussion_r43699104>:
>
> > +    <developers>
> > +        <developer>
> > +            <email>yjxiao@gmail.com</email>
> > +            <name>Yunong Xiao</name>
> > +            <organization>Joyent</organization>
> > +            <organizationUrl>joyent.com</organizationUrl>
> > +            <url>yunong.ca</url>
> > +            <id>yunong</id>
> > +        </developer>
> > +        <developer>
> > +            <email>elijah@zupancic.name</email>
> > +            <name>Elijah Zuoancic</name>
> > +            <organization>Joyent</organization>
> > +            <organizationUrl>joyent.com</organizationUrl>
> > +        </developer>
> > +    </developers>
>
> Oh, yes. We should also probably move Yunong from developers section to a
> contributors section. I believe the convention is that developers actively
> communicate and field questions to the community about the project, whereas
> contributors are people who should not be contracts directly. If he's no
> longer actively involved he should be moved to a contributors section.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/joyent/java-manta/pull/28/files#r43699104>.
>



-- 
-Elijah

**Pull request merged**